templates/chart/base.twig

```diff
@@ -84,7 +84,7 @@ dw.backend.ready(function() {
         $('i#copying-chart').addClass('fa-spinner fa-spin');
         $.ajax({
             type: "POST",
-            url: '/api/charts/'+chart.get('id')+'/copy',
+            url: window.location.protocol + '//' + dw.backend.__api_domain + '/v3/charts/'+chart.get('id')+'/copy',
```

Comment:
the `content-type` is supposed to describe the content type of the request body, but there is no request body in these requests. it's needed for requests including a body because otherwise Joi will not parse it and the validation will fail.

Reply:
what about the `crossDomain: true`?

Reply:
jQuery parses the domain and defaults `crossDomain` to true if it's a cross-domain request (according to the [docs](https://api.jquery.com/jquery.ajax/)). The purpose of `crossDomain` is actually to force jQuery to act like a cross-domain request for same-domain requests, but we can safely forego it.

Reply:
(i also tested all the different changes in the UI before committing)